vite.config.ts

```diff
@@ -0,0 +1,11 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import loadVersion from 'vite-plugin-package-version';
+
+export default defineConfig({
+  plugins: [react(), loadVersion()],
+  resolve: {
+    // this should only be enabled when pnpm-linking the utah-design-package
+    // dedupe: ['firebase', '@arcgis/core'],
```

Comment:
is there an issue with leaving this active when you aren't linking?

Reply:
Yes. Rollup was throwing errors saying that it was unable to resolve firebase modules.